SUIApp/SUIApp/ContentView.swift

```diff
@@ -7,13 +7,101 @@
 
 import SwiftUI
 
+/// Музыкальный плеер
 struct ContentView: View {
+    
+    // MARK: - Private Constatns
+    
+    private enum Constants {
+        static let startText = "Start"
+        static let stopText = "Stop"
+        static let backText = "Back"
+        static let nextText = "Next"
+        static let downloadText = "Скачать"
+        static let shareText = "Поделиться"
+        static let musicPlayerText = "Музыкальный плеер"
+        static let trackText = "Трек"
+        static let downloadToFolderText = "сохранен в папку загрузки"
+        static let alertTitleText = "Поделиться треком?"
+        static let yesText = "Да"
+        static let noText = "Нет"
+        static let currentTimeNumber: Float = 0.0
+    }
+    
+    // MARK: - Public Property
+    
+    @ObservedObject var player = PlayerViewModel()
+    
     var body: some View {
+        NavigationView {
+            playerStack
+                .navigationTitle(Constants.musicPlayerText)
+        }
+    }
+    
+    // MARK: - Private Property
+    
+    @State private var isPressed = false
+    @State private var isShowAlert = false
+    @State private var isShowActionSheet = false
+    private var playerStack: some View {
```

Comment:
```suggestion
    private var playerStackView: some View {
```
где возвращаем some View  на конце пишем View